### PR TITLE
Update 1.7.7.md

### PR DESCRIPTION
**Pull request by MatShir**
Forgotten hook on the order preview

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by matks — APPROVED** [review]
@MatShir I approve the PR but I will not merge it. Look at the ID. I wont merge this ✝️ 🤞

**Review by matthieu-rolland — APPROVED** [review]

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 1]
```diff
@@ -144,14 +144,23 @@ Deprecated hook                   | New aliased hook          | Modification
...
+
+- on the order view page:
 
 New hook                       | Function
 ------------------------------ |:-----------------------------------------------------------------------------------------------------------------------------------
 **displayAdminOrderSide**      | It is displayed in the side column (similar to displayAdminOrderSideBottom) but right under the customer view
 **displayAdminOrderMain**      | It is displayed in the side column (similar to displayAdminOrderMainBottom) but right under the details view
 **actionGetAdminOrderButtons** | It is used to build a collection of buttons in the order thanks to an `ActionsBarButtonsCollection` which is automatically rendered
 
+- on the order list page:
```
Comment (eternoendless) [inline src/content/1.7/modules/core-updates/1.7.7.md:157]:
```suggestion
These hooks have been added on the order list page:
```

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 2]
```diff
@@ -144,14 +144,23 @@ Deprecated hook                   | New aliased hook          | Modification
 **displayBackOfficeOrderActions** | **displayAdminOrderSide** | Although it has been aliased for convenience, the new hook is not displayed at the same place at all (it is now in the side column), so we suggest to verify how your code integrates into the new layout. If you need to add action buttons, use the new **actionGetAdminOrderButtons** hook
 **displayInvoice**                | **displayAdminOrderTop**  | This hook has the same placement and parameters
 
-These hooks have been added to the view page:
+These hooks have been added:
+
+- on the order view page:
```
Comment (eternoendless) [inline src/content/1.7/modules/core-updates/1.7.7.md:149]:
```suggestion
These hooks have been added on the order view page:
```

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @MatShir

**Pull request merged**